impeller/display_list/image_filter.cc

```diff
@@ -102,6 +103,44 @@ std::shared_ptr<FilterContents> WrapInput(const flutter::DlImageFilter* filter,
           outer_dl_filter.get(),
           FilterInput::Make(WrapInput(inner_dl_filter.get(), input)));
     }
+    case flutter::DlImageFilterType::kRuntimeEffect: {
+      auto fragment_program_filter = filter->asRuntimeEffectFilter();
+      FML_DCHECK(fragment_program_filter);
+      const flutter::DlRuntimeEffectImageFilter* runtime_filter =
+          fragment_program_filter->asRuntimeEffectFilter();
```

Comment:
We already have this `fragment_program_filter`, right?  I think `auto` is just making it hard to see.  The autos here in general aren't helping.

Reply:
Done